src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -20,9 +20,11 @@ import {StyleSheet, View} from 'react-native';
 import MapLabel from './components/MapLabel';
 import MapRoute from './components/MapRoute';
 import {createMapLines, getMapBounds, pointOf} from './utils';
+import {VehiclePosition} from '@atb/api/types/generated/ServiceJourneyVehiclesQuery';
 
 export type TravelDetailsMapScreenParams = {
   legs: MapLeg[];
+  _vehiclePosition?: VehiclePosition;
```

Comment:
Why the underscore prefix? I think can just be scrapped if it isn't used. The map will probably find the vehicle position through its own requests.

Reply:
The idea was to send an initial value to have data until we establish subscription, underscore is because it is not yet in use in the map page. The alternative is to send just the service journey id and provide no initial data, but the idea was that could be an easy way to show position instantly. Adding a todo-comment